AdyenActions/UI/View Controllers/DelegatedAuthentication/DAApprovalViewController.swift

```diff
@@ -24,7 +24,7 @@ internal final class DAApprovalViewController: UIViewController {
         let alertController = UIAlertController(
             title: localizedString(.threeds2DAApprovalRemoveAlertTitle, localizationParameters),
             message: localizedString(.threeds2DAApprovalRemoveAlertDescription, localizationParameters),
-            preferredStyle: .alert
+            preferredStyle: .actionSheet
```

Comment:
⚠️ This might crash on iPad
`.actionSheet` need a `sourceView` provided to the `UIAlertController` otherwise it crashes

Reply:
Awesome catch 🏆 . It WILL crash. I've made the changes to run it well on iPad on the simulator. Lets check with the device tomorrow.

Reply:
We tested it today. Thanks for catching this.